sled-agent/src/sled_agent.rs

```diff
@@ -30,6 +31,9 @@ use std::net::SocketAddrV6;
 use std::process::Command;
 use uuid::Uuid;
 
+use crucible::VolumeConstructionRequest; // XXX why?
+                                         //use sled_agent_client::types::VolumeConstructionRequest;
```

Comment:
I have no idea why this is true. I expected to use `sled_agent_client::types::VolumeConstructionRequest` but the compiler complains.